src/MusicalScore/Graphical/EngravingRules.ts

```diff
@@ -336,6 +341,10 @@ export class EngravingRules {
 
     public static FixStafflineBoundingBox: boolean; // TODO temporary workaround
 
+    /** skyline and bottom-line calculation preferences */
+    public PreferredSkyBottomLineBatchCalculatorBackend: SkyBottomLineBatchCalculatorBackendType;
+    public SkyBottomLineBatchCriteria: number;
```

Comment:
rename to SkyBottomLineBatchSize, add JSDoc description (/**)

Reply:
I think renaming this property to `SkyBottomLineBatchSize` is not a good choice; this is not the number of measures in a batch, this is the minimum number of measures in the entire sheet music where the batch algorithms will be enabled. Is there a better one?

Reply:
```typescript

    /** The skyline and bottom-line calculation algorithm to use. */
    public PreferredSkyBottomLineBatchCalculatorBackend: SkyBottomLineBatchCalculatorBackendType;

    /** The minimum number of measures in the entire sheet music where the skyline and bottom-line batch calculation is enabled. */
    public <the minimum number of measures where the batch algorithm is enabled>: number;
```

Reply:
Ah, ok. See, I didn't understand the variable from its name. So, how about SkyBottomLineBatchMinimumMeasures?

Reply:
I think `SkyBottomLineBatchMinMeasures` is okay. I will make a commit for this.